src/lib/coda_base/ledger.ml

```diff
@@ -13,6 +13,33 @@ module Ledger_inner = struct
 
   module Location_at_depth = Location0
 
+  module Location_binable = struct
+    module Arg = struct
+      type t = Location_at_depth.t =
+        | Generic of Location.Bigstring.Stable.Latest.t
+            [@printer
```

Comment:
This attribute is for `[@@deriving show]`, would be nice to remove it since it's unused.